trace-archiver-hdf5/src/main.rs

```diff
@@ -121,7 +123,28 @@ async fn main() -> Result<()> {
                                     .inc();
                             }
                         }
+                    } else if args.control_topic == Some(msg.topic().to_string()) {
+                        // A message has been received from the control topic.
+                        if run_start_buffer_has_identifier(payload) {
+                            debug!("New run start.");
+                            // Start recording trace data to file.
+                            if file.is_none() {
+                                let filename = PathBuf::from(
```

Comment:
Is the filename just the timestamp in milliseconds? Could something more human readable be used instead (ideally [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)).